Sources/ProjectSpec/Breakpoint.swift

```diff
@@ -0,0 +1,157 @@
+import Foundation
+import XcodeProj
+import JSONUtilities
+
+public struct Breakpoint: Equatable {
+
+    public struct Action: Equatable {
+
+        public var type: XCBreakpointList.BreakpointProxy.BreakpointContent.BreakpointActionProxy.ActionExtensionID
+        public var consoleCommand: String?
+        public var message: String?
+        public var conveyanceType: String?
+        public var command: String?
+        public var arguments: String?
+        public var waitUntilDone: Bool?
+        public var script: String?
+        public var soundName: String?
+
+        public init(type: XCBreakpointList.BreakpointProxy.BreakpointContent.BreakpointActionProxy.ActionExtensionID,
+                    consoleCommand: String? = nil,
+                    message: String? = nil,
+                    conveyanceType: String? = nil,
+                    command: String? = nil,
+                    arguments: String? = nil,
+                    waitUntilDone: Bool? = nil,
+                    script: String? = nil,
+                    soundName: String? = nil) {
+            self.type = type
+            self.consoleCommand = consoleCommand
+            self.message = message
+            self.conveyanceType = conveyanceType
+            self.command = command
+            self.arguments = arguments
+            self.waitUntilDone = waitUntilDone
+            self.script = script
+            self.soundName = soundName
+        }
+    }
+
+    public struct Location: Equatable {
+
+        public init() {}
+    }
+
+    public var type: XCBreakpointList.BreakpointProxy.BreakpointExtensionID
+    public var enabled: Bool
+    public var ignoreCount: String
```

Comment:
Let's change relevant properties to `Int` as per comment on ProjectSpec, both for yaml/json parsing and also for easier Swift usage